com.ibm.wala.ide.tests/build.gradle.kts

```diff
@@ -3,9 +3,10 @@ import org.gradle.api.attributes.VerificationType.VERIFICATION_TYPE_ATTRIBUTE
 
 @Suppress("DSL_SCOPE_VIOLATION") // https://github.com/gradle/gradle/issues/22797
 plugins {
-  alias(libs.plugins.eclipse.mavencentral)
+  `java-library`
```

Comment:
Why do we explicitly need `java-library` here now?

Reply:
I don't know why, but the build fails without this, complaining about something related to `testClasses`. I suspect a plugin ordering problem.